### ISSUE TITLE
in browse mode nvda does not advertise the accounts in the list of accounts to sign in to microsoft teams

### ISSUE THREAD

**Issue by fernando-jose-silva** [nvaccess/nvda#13665]
### Steps to reproduce:
sorry if the steps to reproduce are not perfectly documented, I noticed this problem today after using Teams for a long time.
Sign in with a Teams account
Now log out of this account
now sign in with a different account on teams
Again exit this second account.
Teams should now display a page for you to sign back into Teams.
on this page there is a list with the two accounts already used to join teams previously
In browse mode nvda announces when browsing with the arrows below:
2-item clickable list
another down arrow nvda announces
clickable
i would expect nvda to speak
list with 2 items account name 1 clickable
another down arrow
account 2 clickable
If I put nvda in focus  mode nvda correctly announces the name of the two accounts.
the option
Autofocus mode when focus changes
is activated
In these steps I see 2 problems:
being in browse mode nvda should announce the name of the two accounts in the list
nvda should automatically enter focus mode when it arrives in the list since the option:
Autofocus mode when focus changes
is enabled, or at least nvda should have a help text indicating to the user that he can enter the focus mode in the list to better navigate in case the automatic entry in the focus mode in list is impossible.
### Actual behavior:
In browse mode nvda announces when browsing with the arrows below:
2-item clickable list
another down arrow nvda announces
clickable
### Expected behavior:
In browse mode nvda announces when browsing with the arrows below:
list with 2 items account name 1 clickable
another down arrow
account 2 clickable
#### NVDA installed/portable/running from source:
portable
#### NVDA version:
nvda.exe, NVDA alpha-25349,929d5653

#### Windows version:
10.0.19044.16082
#### Name and version of other software in use when reproducing the issue:
Teams.exe, Microsoft Teams 1.5.00.11163
#### Does the issue still occur after restarting your computer?
yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
no
#### If NVDA add-ons are disabled, is your problem still occurring?
yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
yes
as I don't know if the page I've found in the problem is easy to find, I copy the speech explorer's content when navigating in browse mode through this page:
browse mode
Microsoft logo graphic
People using Teams chart
close button
Major Milestone Title Tier 1 Welcome to Microsoft Teams!
Collaborate with your coworkers or coordinate your social groups in one app.
title level 2 Choose an account to continue
2-item clickable list
clickable
out of list content landmark button Use another account or sign up

log:
still as part of this problem, when i went to press nvda + f1 command to harvest the log from this page, not even one nvda command key worked when the focus was on this page,
did not work
nvda + f1
nvda + n
I got the log by restarting nvda and getting the log in the temp folder nvda old .log
if I remove the focus from this page the nvda commands work.
[logteamslogin.txt](https://github.com/nvaccess/nvda/files/8620885/logteamslogin.txt)

**Comment by feerrenrut** [comment]
This very likely a problem with the implementation of MS Teams, as a starting point please submit feedback to Microsoft to let them know about the problem.

**Comment by fernando-jose-silva** [comment]
Can you please direct me to a place where I can get microsoft's attention, is there a website, email, etc?

**Comment by Adriani90** [comment]
@fernando-jose-silva can you still reproduce this issue?
Otherwise you could describe your problem here for example: https://support.microsoft.com/home/contact?ContactUsExperienceEntryPointAssetId=2d4009e7-1300-4766-87e8-7a217496c3d5

Or you can open the feedback hub app by pressing windows key+f and send your feedback from there.
Here on whis website you also can open your feedback hub:
https://support.microsoft.com/en-us/windows/send-feedback-to-microsoft-with-the-feedback-hub-app-f59187f8-8739-22d6-ba93-f66612949332

**Comment by fernando-jose-silva** [comment]
unfortunately still have this problem, thank you very much. Open a request in the feedback hub.Enviado do meu iPhoneEm 6 de fev. de 2023, à(s) 11:38, Adriani90 ***@***.***> escreveu:﻿
@fernando-jose-silva can you still reproduce this issue?
Otherwise you could describe your problem here for example: https://support.microsoft.com/home/contact?ContactUsExperienceEntryPointAssetId=2d4009e7-1300-4766-87e8-7a217496c3d5
Or you can open the feedback hub app by pressing windows key+f and send your feedback from there.
Here on whis website you also can open your feedback hub:
https://support.microsoft.com/en-us/windows/send-feedback-to-microsoft-with-the-feedback-hub-app-f59187f8-8739-22d6-ba93-f66612949332

**Comment by Adriani90** [comment]
I think this has to do with the type of list that is implemented in Teams. It is not a list like a HTML <ul>, but rather a listbox with items. So NVDA will just show the selected item in browse mode like it does generally with list or combo boxes in browsers.

If you wish another design, you need to discuss with the developers of Microsoft Teams.

I am closing this issue since it is not related to NVDA but rather to the design of the Teams application itself.
